api/bibliotheca.py

```diff
@@ -87,6 +92,34 @@ class BibliothecaAPI(BaseBibliothecaAPI, BaseCirculationAPI):
         [v,k] for k, v in delivery_mechanism_to_internal_format.items()
     )
 
+    def external_integration(self, _db):
+        return self.collection.external_integration
+
+    def _run_self_tests(self, _db):
+        def _count_events():
+            now = datetime.datetime.utcnow()
+            five_minutes_ago = now - datetime.timedelta(minutes=5)
```

Comment:
Is there a difference between `timedelta(minutes=5)` (line 113) and `datetime.timedelta(minutes=5)`?

Reply:
Only because this file imports the same class differently.